packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -245,6 +246,14 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
         Have an idea for a new source? Insert its link below to add it to the
         feed.
       </p>
+      <a
+        className="font-bold underline text-theme-label-link"
```

Comment:
We have a specific style guide and information about it can be found in the styles section. I've attached the screenshot as it might help you if you decide to open more PRs! 🎉 

![image](https://user-images.githubusercontent.com/13744167/162549569-c2c208fa-59f2-4d28-b9e4-6c415c3a4aae.png)

```suggestion
        className="font-bold typo-callout underline text-theme-label-link"
```